gamestonk_terminal/portfolio/portfolio_view.py

```diff
@@ -307,15 +319,27 @@ def generate_pg1(self, report: canvas.Canvas) -> None:
         )
         main_t = portfolio_model.get_main_text(self.returns)
         reportlab_helpers.draw_paragraph(report, main_t, 30, 410, 550, 200)
+        rp = self.returns["return"][-1]
```

Comment:
Similar comment here.

Very difficult to make sense out of rp, b, mar tnr.

I would spell them all out - current_return, beta, market_return, fraction_return.